pkgs/by-name/pr/primesieve/package.nix

```
@@ -1,25 +1,38 @@
 { lib
-, stdenv
-, fetchFromGitHub
 , cmake
+, fetchFromGitHub
+, stdenv
+, primecount
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "primesieve";
   version = "12.1";
 
   src = fetchFromGitHub {
     owner = "kimwalisch";
     repo = "primesieve";
-    rev = "v${version}";
+    rev = "v${finalAttrs.version}";
     hash = "sha256-AHl2GfZ1oJ8ZyjJzvg10AqN7TA7HFZ+qa6N2v51Qa78=";
   };
 
+  # Paranoia with some recent Darwin+CMake incidents on split output
+  outputs = if stdenv.isDarwin
+            then [ "out" ]
+            else [ "out" "dev" "lib" "man" ];
```

```suggestion
  outputs = [ "out" "dev" "lib" "man" ];
```

I guess last time it failed because the darwin build generates a [bundle](https://en.wikipedia.org/wiki/Bundle_(macOS)), but that's not the general case. The output split works fine here.